app/src/main/java/com/uwaterloo/watodo/AddEditTaskActivity.java

```diff
@@ -219,8 +247,61 @@ public void onDateSet(DatePicker view, int year,
 
             datePickerDialog.show();
         }
+        if (view == cancelReminder){
+            counter.cancel();
+            selectNotifyTime1.setChecked(false);
+            selectNotifyTime3.setChecked(false);
+            selectNotifyTime7.setChecked(false);
+            TextView textView=findViewById(R.id.notify);
+            textView.setText("Reminder canceled." );
+        }
+
+        if (view == selectNotifyTime1){
+            counterTime(10, selectNotifyTime1);
+        }
+
+        if (view == selectNotifyTime3){
+            counterTime(30, selectNotifyTime3);
+        }
+
+        if (view == selectNotifyTime7){
+            counterTime(70, selectNotifyTime7);
+        }
+
+    }
+
+    public void counterTime(int time, final RadioButton selectNotifyTime){
+
+        counter = new CountDownTimer(time*1000, 1000) {
+            TextView textView=findViewById(R.id.notify);
+
+            public void onTick(long millisUntilFinished) {
+                selectNotifyTime.setChecked(true);
+                textView.setText("The reminder will be sent in " + millisUntilFinished / 1000 + " seconds.");
+
+            }
+
+            public void onFinish() {
+                selectNotifyTime.setChecked(false);
+                String title = editTextTitle.getText().toString();
+                String description = editTextDescription.getText().toString();
+
+                SimpleDateFormat sdf = new SimpleDateFormat("yyyy.MM.dd G 'at' HH:mm:ss z");
+                String currentDateandTime = "Reminder sent on: ";
+                currentDateandTime = currentDateandTime + sdf.format(new Date());
+                textView.setText(currentDateandTime);
+
+                Notify.create(getApplicationContext())
+                        .setTitle(title)
+                        .setContent(description)
+                        .setColor(R.color.colorPrimary)
```

Comment:
looks good. if you add `.setImportance(Notify.NotificationImportance.MAX)` here it will make sure the notification pops up at the top as a "heads up notification"

![image](https://user-images.githubusercontent.com/6562355/78284030-ba9a4e00-74ec-11ea-8711-558479e3634a.png)

![image](https://user-images.githubusercontent.com/6562355/78284006-afdfb900-74ec-11ea-844d-dc86a9f5924a.png)